modules/gateway/rpc.go

```diff
@@ -10,8 +10,11 @@ import (
 	"github.com/NebulousLabs/Sia/modules"
 )
 
+// rpcID is an 8-byte signature that is added to all RPCs to tell the gatway
+// what to do with the RPC.
 type rpcID [8]byte
 
+// String returns a string representation of an rpcID.
```

Comment:
It would be helpful to mention that empty elements of `rpcID`s will be encoded as spaces.